src/store/media-store.ts

```diff
@@ -75,6 +75,17 @@ export const createSettersSlice: StateCreator<
 			}
 			return { isFullscreen: false };
 		}),
+	setIsFullscreen: (isFullscreen: boolean) =>
+		set(state => {
+			if (state.isFullscreen !== isFullscreen) {
+				if (isFullscreen) {
```

Comment:
The same event is emitted independently of `isFullscreen`. Is this intentionally or should this be sth like `fullscreenLeave`? If intentionally: no need for the `if`

Reply:
it is intentionally, because some components that are binded to EventsEmitter are klistening for 2 events:
`fullscreenExit` and `fullscreenEnter`.
there was a typo )